templates/_layouts/init.twig

```diff
@@ -44,6 +44,17 @@
   type: type == 'module' ? type : false,
 }) %}
 
+{# Add external stylesheets.
```

Comment:
Move this section right below the regular `styles` so that all CSS stylesheets are defined in one area.

Reply:
Done.